<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="AWS-PVDI-25" failures="1" tests="1" name="Runner.TestApplication" time="27.427" errors="0" timestamp="2023-11-03T09:49:35 IST" skipped="0">
  <testcase classname="Runner.TestApplication" name="runScenario" time="27.427">
    <failure message="null" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.fail(Assert.java:106)
at stepdefinitions.Murabha_AppDataEntery_CustomerFinancials.user_check_employer_phone_extension_field(Murabha_AppDataEntery_CustomerFinancials.java:138)
at âœ½.User_607 Check Employer Phone Extension Field(file:///C:/Users/inindc00607/git/ULS_Ijarah_1/Gowtham_ULS_New/src/test/java/features/Murabha_AppDataEntery_CustomerFinancials.feature:23)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- Runner.TestApplication -->
